content/en/opentelemetry/agent/_index.md

```diff
@@ -11,6 +11,10 @@ further_reading:
   The Datadog Agent with embedded OpenTelemetry Collector is in Preview. To request access, fill out this form.
 {{< /callout >}} 
 
+{{< site-region region="gov" >}}
+<div class="alert alert-danger"><strong>The Datadog Agent with embedded OpenTelemetry Collector is not FedRAMP or FIPS compliant.</strong><br> FedRAMP customers should not use the embedded OpenTelemetry Collector and should continue using standard Agent deployments.</div>
```

Comment:
"FedRAMP customers should not use the embedded OpenTelemetry Collector and should continue using standard Agent deployments."

I would rephrase this to be: 
FedRAMP customers should not enable or use the embedded OpenTelemetry Collector. 

This is because Otel Collector is part of the Agent. If we refer it the Agent without Otel Collector as "standard Agent" , it creates a misunderstanding that Agent with Otel is a different flavor of Agent. Within 1 or 2 quarters, all Agent image will be revamped.